docs/UserGuide.md

```diff
@@ -454,7 +454,9 @@ Format: **`import`** `FILE_NAME.json`
 * File must be located in the `./data` directory in relative to the directory of `Socius.jar`.
 
 Examples:
-* **`import`** `amy.json` import a contact book via a file named `amy.json` into your existing application.
+* **`import`** `import.json` imports a contact book via a file named `import.json` into your existing application.
+![Folder Directory for imported files](images/ImportExample.png)
```

Comment:
Helpful graphics. Do you think it will be helpful to add a Windows & Linux Version of it?

Reply:
Yeah that would be better, perhaps anyone with windows/linux can add a screenshot for this?